src/dbnode/storage/block/block.go

```diff
@@ -263,13 +263,6 @@ func (b *dbBlock) HasMergeTarget() bool {
 	return hasMergeTarget
 }
 
-func (b *dbBlock) IsRetrieved() bool {
```

Comment:
Thank god, I never understood this. Does this mean you can remove the retriever field from the block as well as any logic dependent on that being present?

The following functions still seem to be using the retriever field:

```
streamWithRLock
resetNewBlockStartWithLock
resetRetrievableWithLock
```

We might even get a small perf boost cause this is will be million of less pointers for the G.C to chase